 Â  INFO 2025-01-09 07:20:22,497 pyk.kast.outer - Loading JSON definition: /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/compiled.json
INFO 2025-01-09 07:20:22,581 pyk.kast.outer - Converting JSON definition to Kast: /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/compiled.json
INFO 2025-01-09 07:20:22,990 pyk.utils - Making directory: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/kcfg
INFO 2025-01-09 07:20:22,990 pyk.utils - Making directory: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/kcfg/nodes
INFO 2025-01-09 07:20:22,994 pyk.kore.rpc - Starting KoreServer: kore-rpc-booster /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/definition.kore --module EDSL --server-port 0 --llvm-backend-library /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/llvm-library/interpreter.so --interim-simplification 25
INFO 2025-01-09 07:20:23,000 pyk.kore.rpc - [PID=666580][stde] [proxy] Loading definition from /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/definition.kore, main module "EDSL"
INFO 2025-01-09 07:20:24,002 pyk.kore.rpc - [PID=666580][stde] [kore][info] Reading the input file TimeSpec {sec = 0, nsec = 24159631}
INFO 2025-01-09 07:20:25,003 pyk.kore.rpc - [PID=666580][stde] [kore][info] Parsing the file TimeSpec {sec = 0, nsec = 110}
INFO 2025-01-09 07:20:26,004 pyk.kore.rpc - [PID=666580][stde] [kore][info] Verifying the definition TimeSpec {sec = 0, nsec = 230}
INFO 2025-01-09 07:20:27,005 pyk.kore.rpc - [PID=666580][stde] [kore][info] Executing TimeSpec {sec = 0, nsec = 707814144}
INFO 2025-01-09 07:20:27,005 pyk.kore.rpc - [PID=666580][stde] [kore][info] Reading the input file TimeSpec {sec = 0, nsec = 24515185}
INFO 2025-01-09 07:20:27,005 pyk.kore.rpc - [PID=666580][stde] [kore][info] Parsing the file TimeSpec {sec = 0, nsec = 120}
INFO 2025-01-09 07:20:27,673 pyk.kore.rpc - KoreServer started: 0.0.0.0:39309, pid=666580
INFO 2025-01-09 07:20:27,673 pyk.kore.rpc - Connecting to host: localhost:39309
INFO 2025-01-09 07:20:27,674 pyk.kore.rpc - Connected to host: localhost:39309
INFO 2025-01-09 07:20:27,674 kevm_pyk.utils - Computing definedness constraint for initial node: ADD_SPEC
INFO 2025-01-09 07:20:27,687 pyk.kore.rpc - Sending request to localhost:39309: 139792892662480-001 - simplify
INFO 2025-01-09 07:20:28,006 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792892662480-001
INFO 2025-01-09 07:20:28,006 pyk.kore.rpc - [PID=666580][stde] [kore][info] Verifying the definition TimeSpec {sec = 0, nsec = 210}
INFO 2025-01-09 07:20:28,006 pyk.kore.rpc - [PID=666580][stde] [proxy] Starting RPC server
INFO 2025-01-09 07:20:28,232 pyk.kore.rpc - Received response from localhost:39309: 139792892662480-001 - simplify
INFO 2025-01-09 07:20:28,245 pyk.kore.rpc - Sending request to localhost:39309: 139792892662480-002 - simplify
INFO 2025-01-09 07:20:28,780 pyk.kore.rpc - Received response from localhost:39309: 139792892662480-002 - simplify
INFO 2025-01-09 07:20:28,781 kevm_pyk.utils - Simplifying initial and target node: ADD_SPEC
INFO 2025-01-09 07:20:28,790 pyk.kore.rpc - Sending request to localhost:39309: 139792892662480-003 - simplify
INFO 2025-01-09 07:20:29,007 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792892662480-002
INFO 2025-01-09 07:20:29,007 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792892662480-003
INFO 2025-01-09 07:20:29,415 pyk.kore.rpc - Received response from localhost:39309: 139792892662480-003 - simplify
INFO 2025-01-09 07:20:29,422 pyk.kore.rpc - Sending request to localhost:39309: 139792892662480-004 - simplify
INFO 2025-01-09 07:20:29,963 pyk.kore.rpc - Received response from localhost:39309: 139792892662480-004 - simplify
INFO 2025-01-09 07:20:29,967 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:20:29,968 pyk.kore.rpc - Connecting to host: localhost:39309
INFO 2025-01-09 07:20:29,968 pyk.kore.rpc - Connected to host: localhost:39309
INFO 2025-01-09 07:20:29,968 pyk.proof.proof - [ZJH] prover type: <class 'pyk.proof.reachability.APRProver'>
INFO 2025-01-09 07:20:29,968 pyk.proof.reachability - [ZJH] init proof
INFO 2025-01-09 07:20:29,971 pyk.kore.rpc - Sending request to localhost:39309: 139792900265744-001 - add-module
INFO 2025-01-09 07:20:30,008 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792900265744-001
INFO 2025-01-09 07:20:30,008 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792892662480-004
INFO 2025-01-09 07:20:30,766 pyk.kore.rpc - Received response from localhost:39309: 139792900265744-001 - add-module
INFO 2025-01-09 07:20:30,781 pyk.kore.rpc - Sending request to localhost:39309: 139792900265744-002 - add-module
INFO 2025-01-09 07:20:31,009 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792900265744-002
INFO 2025-01-09 07:20:31,573 pyk.kore.rpc - Received response from localhost:39309: 139792900265744-002 - add-module
INFO 2025-01-09 07:20:31,573 pyk.proof.reachability - [ZJH] pending: [1]
INFO 2025-01-09 07:20:31,574 pyk.proof.reachability - [ZJH] node_id: 1
INFO 2025-01-09 07:20:31,574 pyk.proof.reachability - [ZJH] is_terminal: False
INFO 2025-01-09 07:20:31,574 pyk.proof.reachability - [ZJH] node_id: 2
INFO 2025-01-09 07:20:31,575 pyk.proof.reachability - [ZJH] is_terminal: False
INFO 2025-01-09 07:20:31,575 pyk.proof.reachability - [ZJH] pending: [1]
INFO 2025-01-09 07:20:31,575 pyk.proof.proof - [ZJH] proof initialized
INFO 2025-01-09 07:20:31,575 pyk.proof.proof - [ZJH] pending: [1]
INFO 2025-01-09 07:20:31,575 pyk.proof.reachability - Before appending step: 1
INFO 2025-01-09 07:20:31,576 pyk.kore.rpc - Connecting to host: localhost:39309
INFO 2025-01-09 07:20:31,576 pyk.proof.proof - Submitted steps for proof: ADD_SPEC
INFO 2025-01-09 07:20:31,576 pyk.kore.rpc - Connected to host: localhost:39309
INFO 2025-01-09 07:20:31,606 pyk.kore.rpc - Sending request to localhost:39309: 139792891217104-001 - implies
INFO 2025-01-09 07:20:32,047 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891217104-001
INFO 2025-01-09 07:20:32,204 pyk.kore.rpc - Received response from localhost:39309: 139792891217104-001 - implies
INFO 2025-01-09 07:20:32,211 pyk.kore.rpc - Sending request to localhost:39309: 139792891217104-002 - execute
INFO 2025-01-09 07:20:33,049 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891217104-002
INFO 2025-01-09 07:20:36,398 pyk.kore.rpc - Received response from localhost:39309: 139792891217104-002 - execute
INFO 2025-01-09 07:20:36,483 pyk.kcfg.kcfg - Extending current KCFG with the following: 4 branches: 1 --> [3, 4, 5, 6]: ['#sizeWordStack ( WS:WordStack , 0 ) <=Int 1023 andBool #if _USEGAS_CELL:Bool #then Gverylow < _SCHEDULE_CELL:Schedule > <=Gas _GAS_CELL:Gas #else true #fi andBool _WORDSTACK_CELL:WordStack ==K W0:Int : W1:Int : WS:WordStack', '#sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) <Int 2', '#sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) +Int -1 >Int 1024', 'notBool #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) +Int -1 >Int 1024 andBool notBool #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) <Int 2']
INFO 2025-01-09 07:20:36,484 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:20:36,490 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:20:36,490 pyk.proof.reachability - Before appending step: 4
INFO 2025-01-09 07:20:36,490 pyk.proof.reachability - Before appending step: 5
INFO 2025-01-09 07:20:36,490 pyk.proof.reachability - Before appending step: 6
INFO 2025-01-09 07:20:36,496 pyk.kore.rpc - Connecting to host: localhost:39309
INFO 2025-01-09 07:20:36,502 pyk.kore.rpc - Connecting to host: localhost:39309
INFO 2025-01-09 07:20:36,508 pyk.kore.rpc - Connecting to host: localhost:39309
INFO 2025-01-09 07:20:36,513 pyk.kore.rpc - Connected to host: localhost:39309
INFO 2025-01-09 07:20:36,528 pyk.kore.rpc - Sending request to localhost:39309: 139792891217104-003 - implies
INFO 2025-01-09 07:20:36,528 pyk.kore.rpc - Connected to host: localhost:39309
INFO 2025-01-09 07:20:36,528 pyk.kore.rpc - Connected to host: localhost:39309
INFO 2025-01-09 07:20:36,553 pyk.kore.rpc - Sending request to localhost:39309: 139792891476496-001 - implies
INFO 2025-01-09 07:20:36,553 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891217104-003
INFO 2025-01-09 07:20:36,619 pyk.kore.rpc - Sending request to localhost:39309: 139792891468176-001 - implies
INFO 2025-01-09 07:20:36,625 pyk.kore.rpc - Sending request to localhost:39309: 139792890688208-001 - implies
INFO 2025-01-09 07:20:37,137 pyk.kore.rpc - Received response from localhost:39309: 139792891217104-003 - implies
INFO 2025-01-09 07:20:37,145 pyk.kore.rpc - Sending request to localhost:39309: 139792891217104-004 - execute
INFO 2025-01-09 07:20:37,209 pyk.kore.rpc - Received response from localhost:39309: 139792890688208-001 - implies
INFO 2025-01-09 07:20:37,217 pyk.kore.rpc - Sending request to localhost:39309: 139792890688208-002 - execute
INFO 2025-01-09 07:20:37,217 pyk.kore.rpc - Received response from localhost:39309: 139792891476496-001 - implies
INFO 2025-01-09 07:20:37,224 pyk.kore.rpc - Sending request to localhost:39309: 139792891476496-002 - execute
INFO 2025-01-09 07:20:37,248 pyk.kore.rpc - Received response from localhost:39309: 139792891468176-001 - implies
INFO 2025-01-09 07:20:37,255 pyk.kore.rpc - Sending request to localhost:39309: 139792891468176-002 - execute
INFO 2025-01-09 07:20:37,551 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891468176-002
INFO 2025-01-09 07:20:37,551 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891476496-001
INFO 2025-01-09 07:20:37,551 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792890688208-002
INFO 2025-01-09 07:20:37,551 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792890688208-001
INFO 2025-01-09 07:20:37,551 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891217104-004
INFO 2025-01-09 07:20:37,551 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891468176-001
INFO 2025-01-09 07:20:37,551 pyk.kore.rpc - [PID=666580][stde] [proxy] Processing request 139792891476496-002
INFO 2025-01-09 07:20:39,406 pyk.kore.rpc - Received response from localhost:39309: 139792891217104-004 - execute
INFO 2025-01-09 07:20:41,246 pyk.kore.rpc - Received response from localhost:39309: 139792890688208-002 - execute
INFO 2025-01-09 07:20:44,254 pyk.kore.rpc - Received response from localhost:39309: 139792891468176-002 - execute
INFO 2025-01-09 07:20:44,280 pyk.kore.rpc - Received response from localhost:39309: 139792891476496-002 - execute
ERROR 2025-01-09 07:20:44,304 kevm_pyk.utils - Proof crashed: ADD_SPEC
Runtime error | code: -32002 | data: {'context': 'CallStack (from HasCallStack):\n  error, called at src/Kore/Rewrite/Function/Evaluator.hs:377:6 in kore-0.1.104-CWw3vBaRpxI3Spyxy9LUQ8:Kore.Rewrite.Function.Evaluator', 'error': 'Error: missing hook\nSymbol\n    LblisValidPoint\'LParUndsRParUnds\'KRYPTO\'Unds\'Bool\'Unds\'G1Point{}\ndeclared with attribute\n    hook{}("KRYPTO.bn128valid")\nWe don\'t recognize that hook and it was not given any rules.\nPlease open a feature request at\n    https://github.com/runtimeverification/haskell-backend/issues\nand include the text of this message.\nWorkaround: Give rules for LblisValidPoint\'LParUndsRParUnds\'KRYPTO\'Unds\'Bool\'Unds\'G1Point{}'}
Traceback (most recent call last):
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 926, in _request
    return self._client.request(method, **params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 246, in request
    return self._default_client.request(method, **params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 325, in request
    self._check(data)
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 336, in _check
    raise JsonRpcError(**response['error'])
pyk.kore.rpc.JsonRpcError: Runtime error

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/utils.py", line 152, in run_prover
    parallel_advance_proof(
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/proof/proof.py", line 386, in parallel_advance_proof
    proof_results = future.result()
                    ^^^^^^^^^^^^^^^
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/proof/proof.py", line 465, in step
    return prover.step_proof(proof_step)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/proof/reachability.py", line 897, in step_proof
    extend_results = self.kcfg_explore.extend_cterm(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kcfg/explore.py", line 231, in extend_cterm
    cterm, next_states, depth, vacuous, next_node_logs = self.cterm_symbolic.execute(
                                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/cterm/symbolic.py", line 110, in execute
    response = self._kore_client.execute(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 987, in execute
    result = self._request('execute', **params)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 928, in _request
    raise self._error(err) from err
pyk.kore.rpc.DefaultError: Runtime error | code: -32002 | data: {'context': 'CallStack (from HasCallStack):\n  error, called at src/Kore/Rewrite/Function/Evaluator.hs:377:6 in kore-0.1.104-CWw3vBaRpxI3Spyxy9LUQ8:Kore.Rewrite.Function.Evaluator', 'error': 'Error: missing hook\nSymbol\n    LblisValidPoint\'LParUndsRParUnds\'KRYPTO\'Unds\'Bool\'Unds\'G1Point{}\ndeclared with attribute\n    hook{}("KRYPTO.bn128valid")\nWe don\'t recognize that hook and it was not given any rules.\nPlease open a feature request at\n    https://github.com/runtimeverification/haskell-backend/issues\nand include the text of this message.\nWorkaround: Give rules for LblisValidPoint\'LParUndsRParUnds\'KRYPTO\'Unds\'Bool\'Unds\'G1Point{}'}
INFO 2025-01-09 07:20:44,305 pyk.kore.rpc - Stopping KoreServer: 0.0.0.0:39309, pid=666580
INFO 2025-01-09 07:20:44,368 pyk.kore.rpc - KoreServer stopped: 0.0.0.0:39309, pid=666580
INFO 2025-01-09 07:20:44,571 pyk.utils - Making directory: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/summaries/ADD_SPEC
Proof timing ADD_SPEC: 14.400331735610962s
INFO 2025-01-09 07:55:45,800 pyk.kast.outer - Loading JSON definition: /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/compiled.json
INFO 2025-01-09 07:55:45,880 pyk.kast.outer - Converting JSON definition to Kast: /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/compiled.json
INFO 2025-01-09 07:55:46,272 pyk.kore.rpc - Starting KoreServer: kore-rpc-booster /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/definition.kore --module EDSL --server-port 0 --llvm-backend-library /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/llvm-library/interpreter.so --interim-simplification 25
INFO 2025-01-09 07:55:46,278 pyk.kore.rpc - [PID=692183][stde] [proxy] Loading definition from /home/zhaoji/.cache/kdist-9ffc41e/evm-semantics/haskell/definition.kore, main module "EDSL"
INFO 2025-01-09 07:55:47,279 pyk.kore.rpc - [PID=692183][stde] [kore][info] Reading the input file TimeSpec {sec = 0, nsec = 23961998}
INFO 2025-01-09 07:55:48,280 pyk.kore.rpc - [PID=692183][stde] [kore][info] Parsing the file TimeSpec {sec = 0, nsec = 120}
INFO 2025-01-09 07:55:49,306 pyk.kore.rpc - [PID=692183][stde] [kore][info] Executing TimeSpec {sec = 0, nsec = 707631302}
INFO 2025-01-09 07:55:49,306 pyk.kore.rpc - [PID=692183][stde] [kore][info] Verifying the definition TimeSpec {sec = 0, nsec = 210}
INFO 2025-01-09 07:55:50,308 pyk.kore.rpc - [PID=692183][stde] [kore][info] Reading the input file TimeSpec {sec = 0, nsec = 25398734}
INFO 2025-01-09 07:55:50,308 pyk.kore.rpc - [PID=692183][stde] [kore][info] Parsing the file TimeSpec {sec = 0, nsec = 110}
INFO 2025-01-09 07:55:50,864 pyk.kore.rpc - KoreServer started: 0.0.0.0:38421, pid=692183
INFO 2025-01-09 07:55:50,865 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:55:50,865 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:55:50,865 kevm_pyk.utils - Computing definedness constraint for initial node: ADD_SPEC
INFO 2025-01-09 07:55:50,879 pyk.kore.rpc - Sending request to localhost:38421: 139736914875792-001 - simplify
INFO 2025-01-09 07:55:51,310 pyk.kore.rpc - [PID=692183][stde] [kore][info] Verifying the definition TimeSpec {sec = 0, nsec = 211}
INFO 2025-01-09 07:55:51,310 pyk.kore.rpc - [PID=692183][stde] [proxy] Starting RPC server
INFO 2025-01-09 07:55:51,310 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736914875792-001
INFO 2025-01-09 07:55:51,450 pyk.kore.rpc - Received response from localhost:38421: 139736914875792-001 - simplify
INFO 2025-01-09 07:55:51,462 pyk.kore.rpc - Sending request to localhost:38421: 139736914875792-002 - simplify
INFO 2025-01-09 07:55:52,004 pyk.kore.rpc - Received response from localhost:38421: 139736914875792-002 - simplify
INFO 2025-01-09 07:55:52,004 kevm_pyk.utils - Simplifying initial and target node: ADD_SPEC
INFO 2025-01-09 07:55:52,013 pyk.kore.rpc - Sending request to localhost:38421: 139736914875792-003 - simplify
INFO 2025-01-09 07:55:52,311 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736914875792-003
INFO 2025-01-09 07:55:52,311 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736914875792-002
INFO 2025-01-09 07:55:52,552 pyk.kore.rpc - Received response from localhost:38421: 139736914875792-003 - simplify
INFO 2025-01-09 07:55:52,564 pyk.kore.rpc - Sending request to localhost:38421: 139736914875792-004 - simplify
INFO 2025-01-09 07:55:53,160 pyk.kore.rpc - Received response from localhost:38421: 139736914875792-004 - simplify
INFO 2025-01-09 07:55:53,166 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:55:53,168 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:55:53,168 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:55:53,169 pyk.proof.proof - [ZJH] prover type: <class 'pyk.proof.reachability.APRProver'>
INFO 2025-01-09 07:55:53,169 pyk.proof.reachability - [ZJH] init proof
INFO 2025-01-09 07:55:53,175 pyk.kore.rpc - Sending request to localhost:38421: 139736914874320-001 - add-module
INFO 2025-01-09 07:55:53,312 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736914874320-001
INFO 2025-01-09 07:55:53,312 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736914875792-004
INFO 2025-01-09 07:55:53,984 pyk.kore.rpc - Received response from localhost:38421: 139736914874320-001 - add-module
INFO 2025-01-09 07:55:54,011 pyk.kore.rpc - Sending request to localhost:38421: 139736914874320-002 - add-module
INFO 2025-01-09 07:55:54,313 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736914874320-002
INFO 2025-01-09 07:55:54,826 pyk.kore.rpc - Received response from localhost:38421: 139736914874320-002 - add-module
INFO 2025-01-09 07:55:54,826 pyk.proof.reachability - [ZJH] pending: [1]
INFO 2025-01-09 07:55:54,826 pyk.proof.reachability - [ZJH] node_id: 1
INFO 2025-01-09 07:55:54,827 pyk.proof.reachability - [ZJH] is_terminal: False
INFO 2025-01-09 07:55:54,827 pyk.proof.reachability - [ZJH] node_id: 2
INFO 2025-01-09 07:55:54,827 pyk.proof.reachability - [ZJH] is_terminal: False
INFO 2025-01-09 07:55:54,827 pyk.proof.reachability - [ZJH] pending: [1]
INFO 2025-01-09 07:55:54,827 pyk.proof.proof - [ZJH] proof initialized
INFO 2025-01-09 07:55:54,827 pyk.proof.proof - [ZJH] pending: [1]
INFO 2025-01-09 07:55:54,827 pyk.proof.reachability - Before appending step: 1
INFO 2025-01-09 07:55:54,828 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:55:54,828 pyk.proof.proof - Submitted steps for proof: ADD_SPEC
INFO 2025-01-09 07:55:54,828 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:55:54,857 pyk.kore.rpc - Sending request to localhost:38421: 139736913433104-001 - implies
INFO 2025-01-09 07:55:55,314 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913433104-001
INFO 2025-01-09 07:55:55,521 pyk.kore.rpc - Received response from localhost:38421: 139736913433104-001 - implies
INFO 2025-01-09 07:55:55,581 pyk.kore.rpc - Sending request to localhost:38421: 139736913433104-002 - execute
INFO 2025-01-09 07:55:56,315 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913433104-002
INFO 2025-01-09 07:55:59,855 pyk.kore.rpc - Received response from localhost:38421: 139736913433104-002 - execute
INFO 2025-01-09 07:55:59,890 pyk.kcfg.kcfg - Extending current KCFG with the following: 4 branches: 1 --> [3, 4, 5, 6]: ['#sizeWordStack ( WS:WordStack , 0 ) <=Int 1023 andBool #if _USEGAS_CELL:Bool #then Gverylow < _SCHEDULE_CELL:Schedule > <=Gas _GAS_CELL:Gas #else true #fi andBool _WORDSTACK_CELL:WordStack ==K W0:Int : W1:Int : WS:WordStack', '#sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) <Int 2', '#sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) +Int -1 >Int 1024', 'notBool #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) +Int -1 >Int 1024 andBool notBool #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) <Int 2']
INFO 2025-01-09 07:55:59,891 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:55:59,896 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:55:59,896 pyk.proof.reachability - Before appending step: 4
INFO 2025-01-09 07:55:59,897 pyk.proof.reachability - Before appending step: 5
INFO 2025-01-09 07:55:59,897 pyk.proof.reachability - Before appending step: 6
INFO 2025-01-09 07:55:59,903 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:55:59,913 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:55:59,919 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:55:59,929 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:55:59,929 pyk.kore.rpc - Sending request to localhost:38421: 139736913433104-003 - implies
INFO 2025-01-09 07:55:59,924 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:55:59,934 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:55:59,949 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913433104-003
INFO 2025-01-09 07:55:59,959 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-001 - implies
INFO 2025-01-09 07:56:00,010 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-001 - implies
INFO 2025-01-09 07:56:00,025 pyk.kore.rpc - Sending request to localhost:38421: 139736912821648-001 - implies
INFO 2025-01-09 07:56:00,561 pyk.kore.rpc - Received response from localhost:38421: 139736913433104-003 - implies
INFO 2025-01-09 07:56:00,569 pyk.kore.rpc - Sending request to localhost:38421: 139736913433104-004 - execute
INFO 2025-01-09 07:56:00,619 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-001 - implies
INFO 2025-01-09 07:56:00,626 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-002 - execute
INFO 2025-01-09 07:56:00,626 pyk.kore.rpc - Received response from localhost:38421: 139736912821648-001 - implies
INFO 2025-01-09 07:56:00,626 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-001 - implies
INFO 2025-01-09 07:56:00,633 pyk.kore.rpc - Sending request to localhost:38421: 139736912821648-002 - execute
INFO 2025-01-09 07:56:00,640 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-002 - execute
INFO 2025-01-09 07:56:00,948 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-001
INFO 2025-01-09 07:56:00,949 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-002
INFO 2025-01-09 07:56:00,949 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-002
INFO 2025-01-09 07:56:00,949 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-001
INFO 2025-01-09 07:56:00,949 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913433104-004
INFO 2025-01-09 07:56:00,949 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912821648-001
INFO 2025-01-09 07:56:00,949 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912821648-002
INFO 2025-01-09 07:56:04,904 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-002 - execute
INFO 2025-01-09 07:56:04,922 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 2: 6 --> 7
INFO 2025-01-09 07:56:04,922 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:04,924 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:04,924 pyk.proof.reachability - Before appending step: 4
INFO 2025-01-09 07:56:04,924 pyk.proof.reachability - Before appending step: 5
INFO 2025-01-09 07:56:04,924 pyk.proof.reachability - Before appending step: 7
INFO 2025-01-09 07:56:04,956 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-003 - implies
INFO 2025-01-09 07:56:04,958 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-003
INFO 2025-01-09 07:56:05,588 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-003 - implies
INFO 2025-01-09 07:56:05,596 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-004 - execute
INFO 2025-01-09 07:56:05,959 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-004
INFO 2025-01-09 07:56:08,109 pyk.kore.rpc - Received response from localhost:38421: 139736912821648-002 - execute
INFO 2025-01-09 07:56:08,137 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 2: 4 --> 8
INFO 2025-01-09 07:56:08,137 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-002 - execute
INFO 2025-01-09 07:56:08,162 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:08,164 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:08,164 pyk.proof.reachability - Before appending step: 5
INFO 2025-01-09 07:56:08,164 pyk.proof.reachability - Before appending step: 7
INFO 2025-01-09 07:56:08,164 pyk.proof.reachability - Before appending step: 8
INFO 2025-01-09 07:56:08,165 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 2: 5 --> 9
INFO 2025-01-09 07:56:08,171 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:08,172 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:08,172 pyk.proof.reachability - Before appending step: 7
INFO 2025-01-09 07:56:08,172 pyk.proof.reachability - Before appending step: 8
INFO 2025-01-09 07:56:08,173 pyk.proof.reachability - Before appending step: 9
INFO 2025-01-09 07:56:08,174 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:08,175 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:08,176 pyk.proof.reachability - Before appending step: 7
INFO 2025-01-09 07:56:08,176 pyk.proof.reachability - Before appending step: 9
INFO 2025-01-09 07:56:08,177 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:08,178 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:08,178 pyk.proof.reachability - Before appending step: 7
INFO 2025-01-09 07:56:09,293 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-004 - execute
INFO 2025-01-09 07:56:09,303 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 non-deterministic branches: 7 --> [10, 11]
INFO 2025-01-09 07:56:09,303 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:09,306 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:09,306 pyk.proof.reachability - Before appending step: 10
INFO 2025-01-09 07:56:09,306 pyk.proof.reachability - Before appending step: 11
INFO 2025-01-09 07:56:09,368 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-003 - implies
INFO 2025-01-09 07:56:09,369 pyk.kore.rpc - Sending request to localhost:38421: 139736912821648-003 - implies
INFO 2025-01-09 07:56:09,374 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-003
INFO 2025-01-09 07:56:09,374 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912821648-003
INFO 2025-01-09 07:56:09,997 pyk.kore.rpc - Received response from localhost:38421: 139736912821648-003 - implies
INFO 2025-01-09 07:56:10,006 pyk.kore.rpc - Sending request to localhost:38421: 139736912821648-004 - execute
INFO 2025-01-09 07:56:10,110 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-003 - implies
INFO 2025-01-09 07:56:10,196 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-004 - execute
INFO 2025-01-09 07:56:10,375 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912821648-004
INFO 2025-01-09 07:56:10,375 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-004
INFO 2025-01-09 07:56:13,140 pyk.kore.rpc - Received response from localhost:38421: 139736912821648-004 - execute
INFO 2025-01-09 07:56:13,157 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 branches: 10 --> [12, 13]: ['_USEGAS_CELL:Bool', 'notBool _USEGAS_CELL:Bool']
INFO 2025-01-09 07:56:13,158 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:13,160 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:13,161 pyk.proof.reachability - Before appending step: 11
INFO 2025-01-09 07:56:13,161 pyk.proof.reachability - Before appending step: 12
INFO 2025-01-09 07:56:13,161 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:13,224 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-005 - implies
INFO 2025-01-09 07:56:13,225 pyk.kore.rpc - Sending request to localhost:38421: 139736912821648-005 - implies
INFO 2025-01-09 07:56:13,228 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-005
INFO 2025-01-09 07:56:13,900 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-005 - implies
INFO 2025-01-09 07:56:13,909 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-006 - execute
INFO 2025-01-09 07:56:13,920 pyk.kore.rpc - Received response from localhost:38421: 139736912821648-005 - implies
INFO 2025-01-09 07:56:13,929 pyk.kore.rpc - Sending request to localhost:38421: 139736912821648-006 - execute
INFO 2025-01-09 07:56:14,039 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-004 - execute
INFO 2025-01-09 07:56:14,049 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 non-deterministic branches: 11 --> [14, 15]
INFO 2025-01-09 07:56:14,050 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:14,053 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:14,053 pyk.proof.reachability - Before appending step: 12
INFO 2025-01-09 07:56:14,053 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:14,053 pyk.proof.reachability - Before appending step: 14
INFO 2025-01-09 07:56:14,053 pyk.proof.reachability - Before appending step: 15
INFO 2025-01-09 07:56:14,059 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:56:14,084 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:56:14,086 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-005 - implies
INFO 2025-01-09 07:56:14,119 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-001 - implies
INFO 2025-01-09 07:56:14,229 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-005
INFO 2025-01-09 07:56:14,229 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-001
INFO 2025-01-09 07:56:14,229 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912821648-005
INFO 2025-01-09 07:56:14,229 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-006
INFO 2025-01-09 07:56:14,229 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912821648-006
INFO 2025-01-09 07:56:14,770 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-005 - implies
INFO 2025-01-09 07:56:14,779 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-006 - execute
INFO 2025-01-09 07:56:14,809 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-001 - implies
INFO 2025-01-09 07:56:14,818 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-002 - execute
INFO 2025-01-09 07:56:15,230 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-006
INFO 2025-01-09 07:56:15,230 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-002
INFO 2025-01-09 07:56:18,418 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-006 - execute
INFO 2025-01-09 07:56:18,438 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 branches: 14 --> [16, 17]: ['_USEGAS_CELL:Bool', 'notBool _USEGAS_CELL:Bool']
INFO 2025-01-09 07:56:18,438 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:18,441 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:18,441 pyk.proof.reachability - Before appending step: 12
INFO 2025-01-09 07:56:18,442 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:18,442 pyk.proof.reachability - Before appending step: 15
INFO 2025-01-09 07:56:18,442 pyk.proof.reachability - Before appending step: 16
INFO 2025-01-09 07:56:18,442 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:18,448 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:56:18,469 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:56:18,481 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-007 - implies
INFO 2025-01-09 07:56:18,491 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-007
INFO 2025-01-09 07:56:18,509 pyk.kore.rpc - Sending request to localhost:38421: 139736911122064-001 - implies
INFO 2025-01-09 07:56:19,288 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-007 - implies
INFO 2025-01-09 07:56:19,297 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-008 - execute
INFO 2025-01-09 07:56:19,302 pyk.kore.rpc - Received response from localhost:38421: 139736911122064-001 - implies
INFO 2025-01-09 07:56:19,311 pyk.kore.rpc - Sending request to localhost:38421: 139736911122064-002 - execute
INFO 2025-01-09 07:56:19,491 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736911122064-002
INFO 2025-01-09 07:56:19,492 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736911122064-001
INFO 2025-01-09 07:56:19,492 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-008
INFO 2025-01-09 07:56:19,695 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-002 - execute
INFO 2025-01-09 07:56:19,707 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 non-deterministic branches: 15 --> [18, 19]
INFO 2025-01-09 07:56:19,707 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:19,711 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:19,711 pyk.proof.reachability - Before appending step: 12
INFO 2025-01-09 07:56:19,711 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:19,711 pyk.proof.reachability - Before appending step: 16
INFO 2025-01-09 07:56:19,711 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:19,712 pyk.proof.reachability - Before appending step: 18
INFO 2025-01-09 07:56:19,712 pyk.proof.reachability - Before appending step: 19
INFO 2025-01-09 07:56:19,718 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:56:19,744 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:56:19,746 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-003 - implies
INFO 2025-01-09 07:56:19,780 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-001 - implies
INFO 2025-01-09 07:56:19,982 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-006 - execute
INFO 2025-01-09 07:56:19,996 pyk.proof.reachability - Caching next step for edge starting from 12
INFO 2025-01-09 07:56:19,997 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 4: 12 --> 20
INFO 2025-01-09 07:56:19,998 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:20,001 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:20,001 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:20,001 pyk.proof.reachability - Before appending step: 16
INFO 2025-01-09 07:56:20,002 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:20,002 pyk.proof.reachability - Before appending step: 18
INFO 2025-01-09 07:56:20,002 pyk.proof.reachability - Before appending step: 19
INFO 2025-01-09 07:56:20,002 pyk.proof.reachability - Before appending step: 20
INFO 2025-01-09 07:56:20,035 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-007 - implies
INFO 2025-01-09 07:56:20,492 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-003
INFO 2025-01-09 07:56:20,493 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-001
INFO 2025-01-09 07:56:20,493 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-007
INFO 2025-01-09 07:56:20,709 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-003 - implies
INFO 2025-01-09 07:56:20,719 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-004 - execute
INFO 2025-01-09 07:56:20,898 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-001 - implies
INFO 2025-01-09 07:56:20,908 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-002 - execute
INFO 2025-01-09 07:56:21,021 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-007 - implies
INFO 2025-01-09 07:56:21,024 pyk.proof.reachability - Using cached step for edge 12 --> 20
INFO 2025-01-09 07:56:21,027 pyk.kcfg.kcfg - Extending current KCFG with the following: 3 non-deterministic branches: 20 --> [21, 22, 23]
INFO 2025-01-09 07:56:21,027 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:21,031 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:21,032 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:21,032 pyk.proof.reachability - Before appending step: 16
INFO 2025-01-09 07:56:21,032 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:21,032 pyk.proof.reachability - Before appending step: 18
INFO 2025-01-09 07:56:21,033 pyk.proof.reachability - Before appending step: 19
INFO 2025-01-09 07:56:21,033 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:21,033 pyk.proof.reachability - Before appending step: 22
INFO 2025-01-09 07:56:21,033 pyk.proof.reachability - Before appending step: 23
INFO 2025-01-09 07:56:21,047 pyk.kore.rpc - Connecting to host: localhost:38421
INFO 2025-01-09 07:56:21,047 pyk.kore.rpc - Connected to host: localhost:38421
INFO 2025-01-09 07:56:21,205 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-008 - implies
INFO 2025-01-09 07:56:21,207 pyk.kore.rpc - Sending request to localhost:38421: 139736912872848-001 - implies
INFO 2025-01-09 07:56:21,496 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-004
INFO 2025-01-09 07:56:21,496 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-008
INFO 2025-01-09 07:56:21,496 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-002
INFO 2025-01-09 07:56:21,496 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912872848-001
INFO 2025-01-09 07:56:22,337 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-008 - implies
INFO 2025-01-09 07:56:22,347 pyk.kore.rpc - Sending request to localhost:38421: 139736912899408-009 - execute
INFO 2025-01-09 07:56:22,386 pyk.kore.rpc - Received response from localhost:38421: 139736912872848-001 - implies
INFO 2025-01-09 07:56:22,396 pyk.kore.rpc - Sending request to localhost:38421: 139736912872848-002 - execute
INFO 2025-01-09 07:56:22,497 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912899408-009
INFO 2025-01-09 07:56:22,497 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912872848-002
INFO 2025-01-09 07:56:26,131 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-004 - execute
INFO 2025-01-09 07:56:26,157 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 branches: 18 --> [24, 25]: ['_USEGAS_CELL:Bool', 'notBool _USEGAS_CELL:Bool']
INFO 2025-01-09 07:56:26,184 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-005 - implies
INFO 2025-01-09 07:56:26,184 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:26,188 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:26,189 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:26,189 pyk.proof.reachability - Before appending step: 16
INFO 2025-01-09 07:56:26,189 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:26,190 pyk.proof.reachability - Before appending step: 19
INFO 2025-01-09 07:56:26,190 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:26,190 pyk.proof.reachability - Before appending step: 22
INFO 2025-01-09 07:56:26,190 pyk.proof.reachability - Before appending step: 23
INFO 2025-01-09 07:56:26,191 pyk.proof.reachability - Before appending step: 24
INFO 2025-01-09 07:56:26,191 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:26,222 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-005
INFO 2025-01-09 07:56:27,617 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-005 - implies
INFO 2025-01-09 07:56:27,628 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-006 - execute
INFO 2025-01-09 07:56:27,635 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-006
INFO 2025-01-09 07:56:27,735 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-002 - execute
INFO 2025-01-09 07:56:27,753 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 non-deterministic branches: 19 --> [26, 27]
INFO 2025-01-09 07:56:27,759 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:27,780 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:27,780 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:27,780 pyk.proof.reachability - Before appending step: 16
INFO 2025-01-09 07:56:27,781 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:27,781 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:27,781 pyk.proof.reachability - Before appending step: 22
INFO 2025-01-09 07:56:27,781 pyk.proof.reachability - Before appending step: 23
INFO 2025-01-09 07:56:27,782 pyk.proof.reachability - Before appending step: 24
INFO 2025-01-09 07:56:27,782 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:27,782 pyk.proof.reachability - Before appending step: 26
INFO 2025-01-09 07:56:27,783 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:27,792 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-003 - implies
INFO 2025-01-09 07:56:28,696 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-003
INFO 2025-01-09 07:56:28,706 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-008 - execute
INFO 2025-01-09 07:56:28,721 pyk.proof.reachability - Caching next step for edge starting from 16
INFO 2025-01-09 07:56:28,728 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 4: 16 --> 28
INFO 2025-01-09 07:56:28,728 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:28,732 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:28,732 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:28,733 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:28,733 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:28,733 pyk.proof.reachability - Before appending step: 22
INFO 2025-01-09 07:56:28,734 pyk.proof.reachability - Before appending step: 23
INFO 2025-01-09 07:56:28,734 pyk.proof.reachability - Before appending step: 24
INFO 2025-01-09 07:56:28,735 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:28,735 pyk.proof.reachability - Before appending step: 26
INFO 2025-01-09 07:56:28,735 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:28,736 pyk.proof.reachability - Before appending step: 28
INFO 2025-01-09 07:56:28,765 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-009 - implies
INFO 2025-01-09 07:56:28,907 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-003 - implies
INFO 2025-01-09 07:56:28,917 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-004 - execute
INFO 2025-01-09 07:56:29,699 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-004
INFO 2025-01-09 07:56:29,699 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-009
INFO 2025-01-09 07:56:29,963 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-009 - implies
INFO 2025-01-09 07:56:29,973 pyk.kore.rpc - Sending request to localhost:38421: 139736912904080-010 - execute
INFO 2025-01-09 07:56:30,701 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912904080-010
INFO 2025-01-09 07:56:31,411 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-006 - execute
INFO 2025-01-09 07:56:31,420 pyk.kcfg.kcfg - Extending current KCFG with the following: stuck node: 23
INFO 2025-01-09 07:56:31,431 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:31,435 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:31,435 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:31,436 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:31,436 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:31,436 pyk.proof.reachability - Before appending step: 22
INFO 2025-01-09 07:56:31,437 pyk.proof.reachability - Before appending step: 24
INFO 2025-01-09 07:56:31,437 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:31,437 pyk.proof.reachability - Before appending step: 26
INFO 2025-01-09 07:56:31,438 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:31,438 pyk.proof.reachability - Before appending step: 28
INFO 2025-01-09 07:56:31,459 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-007 - implies
INFO 2025-01-09 07:56:31,705 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-007
INFO 2025-01-09 07:56:32,530 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-007 - implies
INFO 2025-01-09 07:56:32,540 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-008 - execute
INFO 2025-01-09 07:56:32,714 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-008
INFO 2025-01-09 07:56:36,174 pyk.kore.rpc - Received response from localhost:38421: 139736912872848-002 - execute
INFO 2025-01-09 07:56:36,205 pyk.proof.reachability - Caching next step for edge starting from 22
INFO 2025-01-09 07:56:36,212 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 4: 22 --> 29
INFO 2025-01-09 07:56:36,212 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:36,216 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:36,216 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:36,216 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:36,217 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:36,217 pyk.proof.reachability - Before appending step: 24
INFO 2025-01-09 07:56:36,217 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:36,218 pyk.proof.reachability - Before appending step: 26
INFO 2025-01-09 07:56:36,218 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:36,219 pyk.proof.reachability - Before appending step: 28
INFO 2025-01-09 07:56:36,219 pyk.proof.reachability - Before appending step: 29
INFO 2025-01-09 07:56:36,251 pyk.kore.rpc - Sending request to localhost:38421: 139736912872848-003 - implies
INFO 2025-01-09 07:56:36,265 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912872848-003
INFO 2025-01-09 07:56:37,338 pyk.kore.rpc - Received response from localhost:38421: 139736912872848-003 - implies
INFO 2025-01-09 07:56:37,348 pyk.kore.rpc - Sending request to localhost:38421: 139736912872848-004 - execute
INFO 2025-01-09 07:56:37,350 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912872848-004
INFO 2025-01-09 07:56:38,330 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-008 - execute
INFO 2025-01-09 07:56:38,351 pyk.kcfg.kcfg - Extending current KCFG with the following: 2 branches: 26 --> [30, 31]: ['_USEGAS_CELL:Bool', 'notBool _USEGAS_CELL:Bool']
INFO 2025-01-09 07:56:38,351 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:38,356 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:38,356 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:38,357 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:38,357 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:38,357 pyk.proof.reachability - Before appending step: 24
INFO 2025-01-09 07:56:38,358 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:38,363 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:38,370 pyk.proof.reachability - Before appending step: 28
INFO 2025-01-09 07:56:38,375 pyk.proof.reachability - Before appending step: 29
INFO 2025-01-09 07:56:38,381 pyk.proof.reachability - Before appending step: 30
INFO 2025-01-09 07:56:38,387 pyk.proof.reachability - Before appending step: 31
INFO 2025-01-09 07:56:38,389 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-009 - implies
INFO 2025-01-09 07:56:38,393 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-009
INFO 2025-01-09 07:56:38,420 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-004 - execute
INFO 2025-01-09 07:56:38,536 pyk.proof.reachability - Caching next step for edge starting from 24
INFO 2025-01-09 07:56:38,542 pyk.kcfg.kcfg - Extending current KCFG with the following: basic block at depth 4: 24 --> 32
INFO 2025-01-09 07:56:38,558 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:38,574 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-005 - implies
INFO 2025-01-09 07:56:38,577 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:38,578 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:38,578 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:38,578 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:38,579 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:38,579 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:38,580 pyk.proof.reachability - Before appending step: 28
INFO 2025-01-09 07:56:38,580 pyk.proof.reachability - Before appending step: 29
INFO 2025-01-09 07:56:38,580 pyk.proof.reachability - Before appending step: 30
INFO 2025-01-09 07:56:38,581 pyk.proof.reachability - Before appending step: 31
INFO 2025-01-09 07:56:38,581 pyk.proof.reachability - Before appending step: 32
INFO 2025-01-09 07:56:38,583 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:38,587 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:38,587 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:38,588 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:38,588 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:38,588 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:38,589 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:38,589 pyk.proof.reachability - Before appending step: 28
INFO 2025-01-09 07:56:38,589 pyk.proof.reachability - Before appending step: 30
INFO 2025-01-09 07:56:38,590 pyk.proof.reachability - Before appending step: 31
INFO 2025-01-09 07:56:38,590 pyk.proof.reachability - Before appending step: 32
INFO 2025-01-09 07:56:39,405 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-005
INFO 2025-01-09 07:56:39,452 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-009 - implies
INFO 2025-01-09 07:56:39,455 pyk.proof.reachability - Using cached step for edge 16 --> 28
INFO 2025-01-09 07:56:39,463 pyk.kcfg.kcfg - Extending current KCFG with the following: 3 non-deterministic branches: 28 --> [33, 34, 35]
INFO 2025-01-09 07:56:39,478 pyk.proof.reachability - Wrote proof data for ADD_SPEC: /home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/proofs/ADD_SPEC/proof.json
INFO 2025-01-09 07:56:39,495 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-010 - implies
INFO 2025-01-09 07:56:39,500 pyk.proof.reachability - Before appending step: 3
INFO 2025-01-09 07:56:39,500 pyk.proof.reachability - Before appending step: 13
INFO 2025-01-09 07:56:39,501 pyk.proof.reachability - Before appending step: 17
INFO 2025-01-09 07:56:39,501 pyk.proof.reachability - Before appending step: 21
INFO 2025-01-09 07:56:39,501 pyk.proof.reachability - Before appending step: 25
INFO 2025-01-09 07:56:39,502 pyk.proof.reachability - Before appending step: 27
INFO 2025-01-09 07:56:39,502 pyk.proof.reachability - Before appending step: 30
INFO 2025-01-09 07:56:39,503 pyk.proof.reachability - Before appending step: 31
INFO 2025-01-09 07:56:39,503 pyk.proof.reachability - Before appending step: 32
INFO 2025-01-09 07:56:39,504 pyk.proof.reachability - Before appending step: 33
INFO 2025-01-09 07:56:39,504 pyk.proof.reachability - Before appending step: 34
INFO 2025-01-09 07:56:39,505 pyk.proof.reachability - Before appending step: 35
INFO 2025-01-09 07:56:39,616 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-005 - implies
INFO 2025-01-09 07:56:39,627 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-006 - execute
INFO 2025-01-09 07:56:40,406 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-006
INFO 2025-01-09 07:56:40,406 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-010
INFO 2025-01-09 07:56:40,885 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-010 - implies
INFO 2025-01-09 07:56:40,896 pyk.kore.rpc - Sending request to localhost:38421: 139736913677584-011 - execute
INFO 2025-01-09 07:56:41,408 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736913677584-011
INFO 2025-01-09 07:56:42,874 pyk.kore.rpc - Received response from localhost:38421: 139736912872848-004 - execute
INFO 2025-01-09 07:56:42,915 pyk.kore.rpc - Sending request to localhost:38421: 139736912872848-005 - implies
INFO 2025-01-09 07:56:42,926 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912872848-005
INFO 2025-01-09 07:56:43,950 pyk.kore.rpc - Received response from localhost:38421: 139736912872848-005 - implies
INFO 2025-01-09 07:56:43,953 pyk.proof.reachability - Using cached step for edge 24 --> 32
INFO 2025-01-09 07:56:43,987 pyk.kore.rpc - Sending request to localhost:38421: 139736912872848-006 - implies
INFO 2025-01-09 07:56:44,007 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912872848-006
INFO 2025-01-09 07:56:45,045 pyk.kore.rpc - Received response from localhost:38421: 139736912872848-006 - implies
INFO 2025-01-09 07:56:45,055 pyk.kore.rpc - Sending request to localhost:38421: 139736912872848-007 - execute
INFO 2025-01-09 07:56:45,063 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912872848-007
INFO 2025-01-09 07:56:50,043 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-006 - execute
INFO 2025-01-09 07:56:50,060 pyk.proof.reachability - Caching next step for edge starting from 30
INFO 2025-01-09 07:56:50,093 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-007 - implies
INFO 2025-01-09 07:56:50,097 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-007
INFO 2025-01-09 07:56:51,395 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-007 - implies
INFO 2025-01-09 07:56:51,405 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-008 - execute
INFO 2025-01-09 07:56:51,407 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-008
INFO 2025-01-09 07:57:06,261 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-008 - execute
INFO 2025-01-09 07:57:06,297 pyk.proof.reachability - Caching next step for edge starting from 34
INFO 2025-01-09 07:57:06,331 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-009 - implies
INFO 2025-01-09 07:57:06,335 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-009
INFO 2025-01-09 07:57:07,567 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-009 - implies
INFO 2025-01-09 07:57:07,578 pyk.kore.rpc - Sending request to localhost:38421: 139736912761552-010 - execute
INFO 2025-01-09 07:57:07,581 pyk.kore.rpc - [PID=692183][stde] [proxy] Processing request 139736912761552-010
INFO 2025-01-09 07:57:11,673 pyk.kore.rpc - Received response from localhost:38421: 139736912761552-010 - execute
INFO 2025-01-09 07:57:17,194 pyk.kore.rpc - Received response from localhost:38421: 139736913433104-004 - execute
INFO 2025-01-09 07:57:36,321 pyk.kore.rpc - Received response from localhost:38421: 139736912821648-006 - execute
INFO 2025-01-09 07:57:38,675 pyk.kore.rpc - Received response from localhost:38421: 139736912899408-009 - execute
INFO 2025-01-09 07:57:44,964 pyk.kore.rpc - Received response from localhost:38421: 139736911122064-002 - execute
INFO 2025-01-09 07:57:54,298 pyk.kore.rpc - Received response from localhost:38421: 139736913677584-011 - execute
INFO 2025-01-09 07:57:54,678 pyk.kore.rpc - Received response from localhost:38421: 139736912904080-010 - execute
INFO 2025-01-09 07:57:56,677 pyk.kore.rpc - Received response from localhost:38421: 139736912872848-007 - execute
ERROR 2025-01-09 07:57:56,677 kevm_pyk.utils - Proof crashed: ADD_SPEC
{ false #Equals #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) +Int -1 >Int 1024 }
#And { false #Equals #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) <Int 2 }
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W1:Int : WS:WordStack } )
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W2:Int : WS0:WordStack } )
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W3:Int : WS1:WordStack } )
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W4:Int : WS2:WordStack } )
#And #Not ( #Exists W0:Int . #Exists W1:Int . #Exists WS:WordStack . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W1:Int : WS:WordStack } )
Traceback (most recent call last):
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 926, in _request
    return self._client.request(method, **params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 246, in request
    return self._default_client.request(method, **params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 325, in request
    self._check(data)
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 336, in _check
    raise JsonRpcError(**response['error'])
pyk.kore.rpc.JsonRpcError: Smt solver error

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/cterm/symbolic.py", line 110, in execute
    response = self._kore_client.execute(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 987, in execute
    result = self._request('execute', **params)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kore/rpc.py", line 928, in _request
    raise self._error(err) from err
pyk.kore.rpc.SmtSolverError: SMT solver error: timeout Pattern: \and{SortBool{}}(\and{SortBool{}}(\equals{SortBool{}, SortBool{}}(\dv{SortBool{}}("false"), Lbl'Unds-GT-'Int'Unds'{}(Lbl'UndsPlus'Int'Unds'{}(LblsizeWordStackAux{}(Var'Unds'WORDSTACK'Unds'CELL : SortWordStack{}, \dv{SortInt{}}("0")), \dv{SortInt{}}("-1")), \dv{SortInt{}}("1024"))), \and{SortBool{}}(\equals{SortBool{}, SortBool{}}(\dv{SortBool{}}("false"), Lbl'Unds-LT-'Int'Unds'{}(LblsizeWordStackAux{}(Var'Unds'WORDSTACK'Unds'CELL : SortWordStack{}, \dv{SortInt{}}("0")), \dv{SortInt{}}("2"))), \not{SortBool{}}(\exists{SortBool{}}(VarW0 : SortInt{}, \equals{SortWordStack{}, SortBool{}}(Var'Unds'WORDSTACK'Unds'CELL : SortWordStack{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW0 : SortInt{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW1 : SortInt{}, VarWS : SortWordStack{}))))))), \and{SortBool{}}(\not{SortBool{}}(\exists{SortBool{}}(VarW0 : SortInt{}, \equals{SortWordStack{}, SortBool{}}(Var'Unds'WORDSTACK'Unds'CELL : SortWordStack{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW0 : SortInt{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW2 : SortInt{}, VarWS0 : SortWordStack{}))))), \and{SortBool{}}(\not{SortBool{}}(\exists{SortBool{}}(VarW0 : SortInt{}, \equals{SortWordStack{}, SortBool{}}(Var'Unds'WORDSTACK'Unds'CELL : SortWordStack{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW0 : SortInt{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW3 : SortInt{}, VarWS1 : SortWordStack{}))))), \and{SortBool{}}(\not{SortBool{}}(\exists{SortBool{}}(VarW0 : SortInt{}, \equals{SortWordStack{}, SortBool{}}(Var'Unds'WORDSTACK'Unds'CELL : SortWordStack{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW0 : SortInt{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW4 : SortInt{}, VarWS2 : SortWordStack{}))))), \not{SortBool{}}(\exists{SortBool{}}(VarW0 : SortInt{}, \exists{SortBool{}}(VarW1 : SortInt{}, \exists{SortBool{}}(VarWS : SortWordStack{}, \equals{SortWordStack{}, SortBool{}}(Var'Unds'WORDSTACK'Unds'CELL : SortWordStack{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW0 : SortInt{}, Lbl'UndsColnUndsUnds'EVM-TYPES'Unds'WordStack'Unds'Int'Unds'WordStack{}(VarW1 : SortInt{}, VarWS : SortWordStack{})))))))))))

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/zhaoji/evm-semantics/kevm-pyk/src/kevm_pyk/utils.py", line 152, in run_prover
    parallel_advance_proof(
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/proof/proof.py", line 386, in parallel_advance_proof
    proof_results = future.result()
                    ^^^^^^^^^^^^^^^
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
  File "/nix/store/7hnr99nxrd2aw6lghybqdmkckq60j6l9-python3-3.11.9/lib/python3.11/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/proof/proof.py", line 465, in step
    return prover.step_proof(proof_step)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/proof/reachability.py", line 897, in step_proof
    extend_results = self.kcfg_explore.extend_cterm(
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/kcfg/explore.py", line 231, in extend_cterm
    cterm, next_states, depth, vacuous, next_node_logs = self.cterm_symbolic.execute(
                                                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/zhaoji/.cache/pypoetry/virtualenvs/kevm-pyk-9eA2lNH8-py3.11/lib/python3.11/site-packages/pyk/cterm/symbolic.py", line 120, in execute
    raise self._smt_solver_error(err) from err
pyk.cterm.symbolic.CTermSMTError: { false #Equals #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) +Int -1 >Int 1024 }
#And { false #Equals #sizeWordStack ( _WORDSTACK_CELL:WordStack , 0 ) <Int 2 }
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W1:Int : WS:WordStack } )
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W2:Int : WS0:WordStack } )
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W3:Int : WS1:WordStack } )
#And #Not ( #Exists W0:Int . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W4:Int : WS2:WordStack } )
#And #Not ( #Exists W0:Int . #Exists W1:Int . #Exists WS:WordStack . { _WORDSTACK_CELL:WordStack #Equals W0:Int : W1:Int : WS:WordStack } )
INFO 2025-01-09 07:57:56,679 pyk.kore.rpc - Stopping KoreServer: 0.0.0.0:38421, pid=692183
INFO 2025-01-09 07:57:56,793 pyk.kore.rpc - KoreServer stopped: 0.0.0.0:38421, pid=692183
Proof timing ADD_SPEC: 123.62447190284729s
